editable-samples/editor/codemirror/lib/codemirror.js

```diff
@@ -1620,7 +1620,7 @@ function extractLineClasses(type, output) {
     var prop = lineClass[1] ? "bgClass" : "textClass"
     if (output[prop] == null)
       { output[prop] = lineClass[2] }
-    else if (!(new RegExp("(?:^|\s)" + lineClass[2] + "(?:$|\s)")).test(output[prop]))
+    else if (!(new RegExp("(?:^|\\s)" + lineClass[2] + "(?:$|\\s)")).test(output[prop]))
```

Comment:
This change is a fix for https://github.com/mdn/js-examples/runs/10603498184

Reply:
This is how it's handled in codemirror5:

https://github.com/codemirror/codemirror5/blob/9e864a1bb7c4c452f462d7f8d8be111c8bb8ad6f/src/line/highlight.js#L201